content/techniques/configuration.md

```diff
@@ -151,9 +151,10 @@ import { join } from 'path';
 const YAML_CONFIG_FILENAME = 'config.yml';
 
 export default () => {
-  return yaml.load(
-    fs.readFileSync(join(__dirname, YAML_CONFIG_FILENAME), 'utf8'),
-  );
+  return (<object>
```

Comment:
Is `object` here another way to say `Record<string, string>`?

Reply:
Honestly, I am not sure.
What I am sure about, that this works fine, while the previous one didn't ^^'

Reply:
But I can try changing it to a cast to Record<string, string>, if you say that's more "idiomatic" ... or simply less dirty

Reply:
If `object` is meant to represent a JSON, it usually is better typed as `Record<string, any>`. If `yaml` parses booleans, strings, and numbers, then definitely keep it as `any` in the second generic parameter. If it all comes back as strings then `Record<string, string>` could be used instead

Reply:
Ok, then i'll change it. (In the example repo as well)

Reply:
> But I can try changing it to a cast to Record<string, string>, if you say that's more "idiomatic" ... or simply less dirty

Yeah, generally I try to stay away from `object` and `function` as types and give things like `Record<K, V>` and `(args: any[]): void => {}` as they're more verbose and have explicit meanings in Typescript. Plus, eslint rules forbidding certain types

Reply:
Ok, changed it. The example repo above should have the update to.